cmd/av/commit_amend.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"emperror.dev/errors"
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/config"
+	"github.com/aviator-co/av/internal/git"
+	"github.com/aviator-co/av/internal/meta"
+	"github.com/aviator-co/av/internal/utils/colors"
+	"github.com/spf13/cobra"
+)
+
+var commitAmendFlags struct {
+	// The commit message to update with.
+	Message string
+
+	// Same as `git commit --amend --no-edit`. Amends a commit without changing its commit message.
+	NoEdit bool
+}
+
+
+var commitAmendCmd = &cobra.Command{
+	Use:   "amend",
+	Short: "amend a commit",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		repo, err := getRepo()
+		if err != nil {
+			return err
+		}
+
+		currentBranchName, err := repo.CurrentBranchName()
+		if err != nil {
+			return errors.WrapIf(err, "failed to determine current branch")
+		}
+
+		if err := commitAmend(repo, currentBranchName, commitAmendFlags); err != nil {
+			return err
+		}
+
+		return nil
+	},
+}
+
+func init() {
+	commitAmendCmd.Flags().StringVarP(&commitAmendFlags.Message, "message", "m", "", "the commit message")
+	commitAmendCmd.Flags().BoolVarP(&commitAmendFlags.NoEdit, "no-edit", "a", false, "amend a commit without changing its commit message")
```

Comment:
Let's not add shorthand versions that don't exist in Git. You can add a non-shorthand flag with just `BoolVar`.